Tools/jit/_targets.py

```diff
@@ -154,6 +157,8 @@ async def _compile(
             "-o",
             f"{o}",
             f"{c}",
+            # Allow user-provided CFLAGS to override any defaults
+            *shlex.split(self.cflags),
             *self.args,
```

Comment:
These are probably best at the *very* end.

```suggestion
            *self.args,
            # Allow user-provided CFLAGS to override any defaults
            *shlex.split(self.cflags),
```